airbyte-webapp/src/components/EntityTable/components/SortButton.module.scss

```diff
@@ -0,0 +1,21 @@
+@use "../../../scss/colors";
+
+.sortButtonView {
+  display: inline-block;
+  align-items: center;
+  justify-content: center;
+  min-width: 18px;
+  font-size: 11px;
```

Comment:
@timroes about `SortButton` (two comments below the same). earlier we used a base button component that only showed icon inside corresponding with sort order. still, for me, the better solution here is to make the table header clickable and get rid of the `<Button />` component here. in this case, I don't need to make some magic with the base <Button /> component, to make it look good, and also it will be easier for users to use sort, without the need to hit precisely small sorting button. what do you think?